Octokit.Tests.Integration/Clients/DeploymentsClientTests.cs

```diff
@@ -17,33 +19,38 @@ public DeploymentsClientTests()
 
         _deploymentsClient = github.Repository.Deployment;
         _context = github.CreateRepositoryContext("public-repo").Result;
+        _commits = new List<Commit>();
 
-        var blob = new NewBlob
+        for (int i = 0; i < 6; i++)
```

Comment:
@dampir while I like making this initial code more generic (and extending it to do multiple commits) I think we can extract this from the constructor and just have a standalone method or methods.

A couple of reasons:

 - we're creating 6 commits for every test run, even though many of them only need one
 - by moving the creation of the commits closer to the test itself, we can make the intent clearer
 - we're doing a lot of `.Result` here, which blocks the current thread as you can't await in a constructor. By moving this async code closer into the test itself, we can leverage async/await better - which the test runner supports